mlir/lib/Conversion/VectorToLLVM/ConvertVectorToLLVM.cpp

```diff
@@ -269,49 +269,32 @@ class VectorGatherOpConversion
     if (failed(isMemRefTypeSupported(memRefType, *this->getTypeConverter())))
       return failure();
 
+    VectorType vType = gather.getVectorType();
+    if (vType.getRank() > 1)
+      return failure();
+
     auto loc = gather->getLoc();
 
     // Resolve alignment.
     unsigned align;
     if (failed(getMemRefAlignment(*getTypeConverter(), memRefType, align)))
       return failure();
 
+    // Resolve address.
```

Comment:
If I'm reading the code right,
```suggestion
    // Resolve base address.
```

Reply:
I'm mainly trying to match exactly the scatter conversion logic. To keep them same, I'm opting for "Resolve address" for now. I've cleaned it up to not have duplicating address resolution comments.